bpaingest/projects/stemcells/ingest.py

```diff
@@ -584,6 +611,7 @@ def _get_resources(self):
                     if not files.proteomics_pool_filename_re.match(filename):
                         raise Exception("unhandled file: %s" % (filename))
                     continue
+                logger.debug("file name is: {0}".format(filename))
                 resource = file_info.copy()
```

Comment:
don't commit debug logs

Reply:
@grahame If we have a main set to INFO, this should be OK. 
Having to comment out and comment in debug logs seems like a painful way to deal with this